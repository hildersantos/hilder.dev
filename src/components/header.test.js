import React from "react"
import { render } from "react-testing-library"

import Header from "./header"

describe("Header", () => {
  it("renders withour errors", () => {
    const { asFragment } = render(<Header siteTitle="Gatsby" />)
    expect(asFragment()).toMatchSnapshot()
  })
})
