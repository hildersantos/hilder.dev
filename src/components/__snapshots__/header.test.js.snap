// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders withour errors 1`] = `
<DocumentFragment>
  <header
    class="header-inner"
  >
    <a
      class="header-inner__link--home header-inner__link"
      href="/"
    >
      Gatsby
    </a>
  </header>
</DocumentFragment>
`;
